commands/host.vegman

```diff
@@ -280,36 +245,14 @@ function cmd_virtualmedia_mount {
     return 1
   fi
 
-  if [[ ! -z ${username} ]] || [[ ! -z ${password} ]]
-  then
-    local cred_pipe_fifo=$(mktemp -u)
-    mkfifo $cred_pipe_fifo
-    exec {cred_pipe}<>${cred_pipe_fifo}
-    rm $cred_pipe_fifo
-
-    printf "%s\0%s\0\n" "${username:0:64}" "${password:0:64}" >&{cred_pipe}
-  fi
-
   if [[ "${mode}" == "legacy" ]]; then
-    busctl call xyz.openbmc_project.VirtualMedia \
-      /xyz/openbmc_project/VirtualMedia/Legacy/USB1 \
-      xyz.openbmc_project.VirtualMedia.Legacy \
-      Mount sbyv "$imagefile" $readwrite ${type} i ${cred_pipe} > /dev/null  && \
-    (echo "Virtual Media (USB1) mounted"; \
-    log_audit "Host virtual media mount : ${imagefile}") || \
-    echo "Failed to mount Virtual Media (USB1)"
+    vm-cli -o mount ${type} ${request_creds} ${readwrite} --${mode} --file ${imagefile} && \
+      log_audit "Host virtual media mount : ${imagefile}" || \
+      log_audit "Failed to mount Virtual Media (USB1)"
```

Comment:
Are you sure that this and similars below message should be put in the audit log instead of user screen?
Look at the previous implementation:
- `log_audit` puts the message to the system logs with corresponding tags.
- `echo` puts the message to the user screen.

Reply:
The user message will be printed by the `vm-cli`.

Reply:
And why have you decided to put the error messages to the audit logs?

Reply:
I believed the old implementation is missed that(error audit message) because in the umount operation we logging both case - success and failure.

Reply:
No, the audit log should contain the facts of settings/mode changing.
The error messages are not relevant to it and will look like a trash.
I guess the previous umount implementation was just misreviewed.

Reply:
fixed